python/lsst/daf/butler/cli/cliLog.py

```diff
@@ -92,7 +110,12 @@ def initLog(cls, longlog):
             pythonLogger.addHandler(cls._lsstLogHandler)
```

Comment:
This line is why you have so much trouble in your tests capturing log messages.  We need to make it much more explicit in documentation (at least for this function) that this is going on. It is there but a sphinx warning might make it clearer. I keep forgetting we do this.  The reason we can't go the other way round is that I have no way to forward the C++ log messages back to python.